modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/validation/util/DateParameterUtil.java

```diff
@@ -37,6 +39,10 @@
  */
 public class DateParameterUtil {
 
+	public static final List<String> datePatterns = Arrays.asList(
```

Comment:
I think `dateTimePatterns` is a better variable name, since they all deal with time, what do you think?